src/huggingface_hub/templates/datasetcard_template.md

```diff
@@ -1,103 +1,130 @@
 ---
-# For reference on model card metadata, see the spec: https://github.com/huggingface/hub-docs/blob/main/datasetcard.md?plain=1
+# For reference on dataset card metadata, see the spec: https://github.com/huggingface/hub-docs/blob/main/datasetcard.md?plain=1
 # Doc / guide: https://huggingface.co/docs/hub/datasets-cards
 {{ card_data }}
 ---
 
 # Dataset Card for {{ pretty_name | default("Dataset Name", true) }}
 
-## Dataset Description
+<!-- Provide a quick summary of the dataset. -->
 
-- **Homepage:** {{ homepage_url | default("", true)}}
-- **Repository:** {{ repo_url | default("", true)}}
-- **Paper:** {{ paper_url | default("", true)}}
-- **Leaderboard:** {{ leaderboard_url | default("", true)}}
-- **Point of Contact:** {{ point_of_contact | default("", true)}}
+{{ dataset_summary | default("", true) }}
 
-### Dataset Summary
+## Dataset Details
 
-{{ dataset_summary | default("[More Information Needed]", true)}}
+### Dataset Description
 
-### Supported Tasks and Leaderboards
+<!-- Provide a longer summary of what this dataset is. -->
 
-{{ supported_tasks_and_leaderboards_section | default("[More Information Needed]", true)}}
+{{ dataset_description | default("", true) }}
 
-### Languages
+- **Curated by:** {{ curators | default("[More Information Needed]", true)}}
+- **Shared by [optional]:** {{ shared_by | default("[More Information Needed]", true)}}
+- **Language(s) (NLP):** {{ language | default("[More Information Needed]", true)}}
+- **License:** {{ license | default("[More Information Needed]", true)}}
 
-{{ languages_section | default("[More Information Needed]", true)}}
+### Dataset Sources [optional]
 
-## Dataset Structure
+<!-- Provide the basic links for the dataset. -->
+
+- **Repository:** {{ repo | default("[More Information Needed]", true)}}
+- **Paper [optional]:** {{ paper | default("[More Information Needed]", true)}}
+- **Demo [optional]:** {{ demo | default("[More Information Needed]", true)}}
+
+## Uses
+
+<!-- Address questions around how the dataset is intended to be used. -->
 
-### Data Instances
+### Out-of-Scope Use
```

Comment:
Model cards also have Direct Use and Downstream Use. Would it make sense to have them here as well ?

I think Direct Use makes sense (partially in replacement of "Supported Tasks" that is removed in this PR too). Downstream use is less clear on the other hand.

Reply:
Yes, `Downstream Use` makes sense for models (it's also optional there), but not so much for datasets IMO.

Having `Direct Use`' info under `Uses` should be OK, no? (my rationale was to simplify the structure a bit here)

Reply:
Yep, I agree. `Direct Use` under `Uses`, but `Downstream Use` as optional (or not there at all -- and I'm leaning towards _not there at all_ for brevity + because that has more to do with a model)